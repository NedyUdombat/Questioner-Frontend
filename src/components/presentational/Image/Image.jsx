import React from 'react';
import PropTypes from 'prop-types';

// styles
import './Image.scss';

const Image = ({ src, alt, className }) => {
  return <img src={src} alt={alt} className={className} />;
};

Image.propTypes = {
  src: PropTypes.string,
  alt: PropTypes.string,
  className: PropTypes.string,
};

export default Image;
