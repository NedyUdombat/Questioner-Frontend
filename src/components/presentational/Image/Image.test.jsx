import React from 'react';
import { mount } from 'enzyme';

import Image from './Image';

describe('Image', () => {
  it('should render correctly', () => {
    const component = mount(
      <Image alt="Image alt" src="http://dummyimage.jpg" className="class" />,
    );
    expect(component).toMatchSnapshot();
  });
});
