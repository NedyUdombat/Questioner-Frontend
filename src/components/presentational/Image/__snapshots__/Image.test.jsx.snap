// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should render correctly 1`] = `
<Image
  alt="Image alt"
  className="class"
  src="http://dummyimage.jpg"
>
  <img
    alt="Image alt"
    className="class"
    src="http://dummyimage.jpg"
  />
</Image>
`;
